src/corehost/cli/args.cpp

```diff
@@ -103,6 +109,16 @@ bool parse_arguments(const pal::string_t& deps_path, const pal::string_t& probe_
     }
 
     pal::getenv(_X("DOTNET_PACKAGES_CACHE"), &args.dotnet_packages_cache);
-    pal::getenv(_X("DOTNET_SERVICING"), &args.dotnet_servicing);
+    pal::getenv(_X("DOTNET_EXTENSIONS"), &args.dotnet_extensions);
```

Comment:
This is not required. Any shared location path will be specified at command-line (as --additionalProbingPath).